app/com/linkedin/drelephant/spark/fetchers/SparkRestClient.scala

```diff
@@ -135,6 +157,31 @@ class SparkRestClient(sparkConf: SparkConf) {
       }
     }
   }
+
+  private def getLogData(attemptTarget: WebTarget): Option[SparkLogDerivedData] = {
```

Comment:
Just a style comment. Maybe you should declare this method before getApplicationLogs because this the caller method.